pkg/util/util.go

```diff
@@ -86,3 +117,10 @@ func MtimeHasher() func(string) (string, error) {
 	}
 	return hasher
 }
+
+// SHA256 returns the shasum of the contents of r
+func SHA256(r io.Reader) (string, error) {
+	hasher := sha256.New()
+	_, err := io.Copy(hasher, r)
```

Comment:
return error if its not nil.
the next line might throw an exception.

Reply:
done

Reply:
done